set.mm

```diff
@@ -11939,23 +11939,23 @@ This definition (in the form of ~ dfifp2 ) appears in Section II.24 of
   Universal quantifier for use by df-tru
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 
-  Even though it isn't ordinarily part of propositional calculus, the universal
-  quantifier ` A. ` is introduced here so that the soundness of definition
-  ~ df-tru can be checked by the same algorithm that is used for predicate
-  calculus.  Its first real use is in definition ~ df-ex in the predicate
-  calculus section below.  For those who want propositional calculus to be
-  self-contained i.e. to use wff variables only, the alternate definition
-  ~ dftru2 may be adopted and this subsection moved down to the start of the
-  subsection with ~ wex below.  However, the use of ~ dftru2 as a definition
-  requires a more elaborate definition checking algorithm that we prefer to
-  avoid.
+  Even though it is not ordinarily part of propositional calculus, the
+  universal quantifier ` A. ` is introduced here so that the soundness of
+  definition ~ df-tru can be checked by the same algorithm that is used for
+  predicate calculus.  Its first real use is in Definition ~ df-ex in the
```

Comment:
Is there a rule/convention when to start "axiom", "theorem", "definition" with a capitel letter and when not? Shall we use capital letters if a reference ("~ <label>") follows? 

I checked the current set.mm and found the following:

* " Axiom ~ " vs. " axiom ~ ": 12 (most of them either at the beginning of a sentence, or as part of the name of a concrete axiom, e.g., "Replacement Axiom ~ funimaex") vs. 72
* " Definition ~ " vs. " definition ~ ": 4 (3 of them at the beginning of a sentence) vs. 55
* " Theorem ~ " vs. " theorem ~ ": 48 (at least 27 of them at the beginning of a sentence) vs. 135

It seems that the lower case versions are usually used.

Reply:
In the math literature, it's definitely the norm to write "by Theorem 1.2", or "see Proposition 2.1 of [Megill]".  Statements in Metamath databases are not numbered but labeled, so it makes sense to me to write "Definition ~ xxx ", following the dominant convention. What do you think ?

If yes, thanks to your search, we see we should make a round 250 changes. Probably in a future PR ?

Reply:
@benjub In principle I agree, but we should hear the opinion of others. Maybe you can create an "issue" for this (and the alignment can be done in a separate PR, as you suggested).

Reply:
I created issue #4163. Since in any case, we'll have to uniformize one way or the other in a future PR, and it was inconsistent already anyway, and since I don't remember which ones I changed in this PR, can we leave it like that?